packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -284,9 +286,7 @@ export class AccountForm extends PureComponent {
 
 AccountForm.propTypes = {
   account: PropTypes.object,
-  fields: PropTypes.object,
-  oauth: PropTypes.object,
-  locales: PropTypes.object,
+  konnector: PropTypes.object.isRequired,
```

Comment:
But also if we're passing a full object, we can describe it's shape here :-)

Reply:
[Same remark](https://github.com/cozy/cozy-libs/pull/275#discussion_r258502959) as @Crash--, you definitely should meet each other.